libvirt.go

```diff
@@ -40,7 +40,8 @@ type Driver struct {
 
 	// Libvirt connection and state
 	connectionString string
-	conn             *libvirt.Connect
+	sessionConn      *libvirt.Connect
+	systemConn       *libvirt.Connect
```

Comment:
dunno if it's worth naming it roSystemConn ?

Reply:
I don't think so. To me, that's a detail.